src/packaging/packages.targets

```diff
@@ -60,14 +60,37 @@
             <ILCompilerSdkFiles Include="bootstrappercpp" />
             <ILCompilerSdkFiles Include="System.Private.CoreLib.Native" Condition="'$(OsEnvironment)'!='Windows_NT'" />
 
+            <!-- ILCompiler.SDK Cpp Codegen support files -->
+            <ILCompilerSdkCppCodegenFiles Include="Native/gc/env/etmdummy.h" />
+            <ILCompilerSdkCppCodegenFiles Include="Native/gc/env/gcenv.base.h" />
```

Comment:
To confirm, post your PR, we will only need sal.h/gcenv.structs.h/gcenv.base.h from gc\env, right (atleast that is what I saw in the PR)?

We will still need the headers form GC, right?